src/core/crash-handler.c

```diff
@@ -155,9 +155,6 @@ _noreturn_ static void crash(int sig, siginfo_t *siginfo, void *context) {
         (void) sigaction(SIGCHLD, &sa, NULL);
 
         if (arg_crash_shell) {
-                log_notice("Executing crash shell in 10s...");
-                (void) sleep(10);
```

Comment:
i am fine with dropping the sleeping here. But I think we should keep a message here that is shown ebfore we fork off the crash shell, just to make clear that this is what happens now.

i.e. keep `log_notice("Executing crash shell...");`

in place